doc/faq.rst

```diff
@@ -181,86 +181,60 @@ Using Spyder
 .. dropdown:: Q: How do I get Spyder to work with my existing Python packages/environment?
    :container: + dropdown-id-using-existing-environment
 
-   To work with an existing environment in Spyder, you need to change Spyder’s default Python interpreter.
+   To work with an existing environment in Spyder, change the default Python interpreter for new :doc:`/panes/ipythonconsole`\s to point to this environment.
    To do so, click the name of the current environment in the status bar, and then click :guilabel:`Change default environment in Preferences`.
 
    .. image:: /images/faq/faq-change-environment.png
```

Comment:
Yeah, the screenshot is a bit out of date with what the latest versions show; specifically, they have two entries, one for the current console env and one for the completion env, but always show the same thing—and even more confusingly, the _Completions_ entry is the one with the popup, and _not_ the main environment entry.

IIRC, it was discussed recently that we should just eliminate the redundant `Completions` entry and re-consolidate on the main environment entry, making the status bar again look like it does in the screenshot—but I'm not sure the status of that. @ccordoba12 ?

Given the situation, I'm thinking maybe it's better to just describe how to get to the Python interpreter settings through the normal preferences route for now, since that is easier to find and more consistent across Python versions?

Reply:
> Given the situation, I'm thinking maybe it's better to just describe how to get to the Python interpreter settings through the normal preferences route for now, since that is easier to find and more consistent across Python versions?

I agree.

Reply:
Thanks, went ahead and implemented that. It also makes the FAQ answer shorter and simpler overall, since it already mentions that pane's in preferences.